pkgs/development/python-modules/darkgraylib/default.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  toml,
+  pygments,
+  pytestCheckHook,
+  pytest-kwparametrize,
+  pkgs,
+}:
```

Comment:
```suggestion
}:

```
nitpick

Reply:
No point in nitpicking if the action does it. If the formatter thinks this is off, then it will fire.